roles/cockpit-ui/tasks/main.yml

```diff
@@ -22,54 +22,50 @@
   changed_when: false
```

Comment:
Can remove everything from here up if you switch the oc_route kubeconfig arguments to use `{{ openshift_master_config_dir }}/admin.kubeconfig }}`